modules/combined/test/tests/gap_heat_transfer_mortar/finite-2d-rz/finite.i

```diff
@@ -0,0 +1,356 @@
+E_block = 1e7
+E_plank = 1e7
+elem = QUAD4
+order = FIRST
+name = 'finite'
+
+[Mesh]
+  patch_size = 80
+  patch_update_strategy = auto
+  [plank]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = 0.6
+    ymin = -10
+    ymax = 10
+    nx = 2
+    ny = 67
+    elem_type = ${elem}
+    boundary_name_prefix = plank
+  []
+  [plank_id]
+    type = SubdomainIDGenerator
+    input = plank
+    subdomain_id = 1
+  []
+
+  [block]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0.61
+    xmax = 1.21
+    ymin = 7.7
+    ymax = 8.5
+    nx = 3
+    ny = 4
+    elem_type = ${elem}
+    boundary_name_prefix = block
+    boundary_id_offset = 10
+  []
+  [block_id]
+    type = SubdomainIDGenerator
+    input = block
+    subdomain_id = 2
+  []
+
+  [combined]
+    type = MeshCollectionGenerator
+    inputs = 'plank_id block_id'
+  []
+  [block_rename]
+    type = RenameBlockGenerator
+    input = combined
+    old_block_id = '1 2'
+    new_block_name = 'plank block'
+  []
+
+  [secondary]
+    input = block_rename
+    type = LowerDBlockFromSidesetGenerator
+    sidesets = 'block_left'
+    new_block_id = '30'
+    new_block_name = 'frictionless_secondary_subdomain'
+  []
+  [primary]
+    input = secondary
+    type = LowerDBlockFromSidesetGenerator
+    sidesets = 'plank_right'
+    new_block_id = '20'
+    new_block_name = 'frictionless_primary_subdomain'
+  []
+[]
+
+[Problem]
+  coord_type = RZ
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+[]
+
+[Variables]
+  [disp_x]
+    order = ${order}
+    block = 'plank block'
+    scaling = ${fparse 2.0 / (E_plank + E_block)}
+  []
+  [disp_y]
+    order = ${order}
+    block = 'plank block'
+    scaling = ${fparse 2.0 / (E_plank + E_block)}
+  []
+  [temp]
+    order = ${order}
+    block = 'plank block'
+    scaling = 1e-1
+  []
+  [thermal_lm]
+    order = ${order}
+    block = 'frictionless_secondary_subdomain'
+    scaling = 1e-7
+  []
+  [frictionless_normal_lm]
+    order = FIRST
+    block = 'frictionless_secondary_subdomain'
+    scaling = 1e3
```

Comment:
Any alternative to the individual scaling of the variables for this problem?

Reply:
We still do not have an automated way of scaling variables that may not have diagonal dependence. I have an open issue for it here: https://github.com/idaholab/moose/issues/16212